tests/test_minimal/test_tools/test_context_tools.py

```diff
@@ -220,3 +221,11 @@ def test_make_or_load_nwbfile_on_corrupt_file(tmpdir: Path) -> None:
     with make_or_load_nwbfile(nwbfile_path=nwbfile_path, nwbfile=nwbfile_in, overwrite=True) as nwbfile:
         time_series = mock_TimeSeries()
         nwbfile.add_acquisition(time_series)
+
+
+def test_raise_error_when_metadata_is_empty_and_creation_is_needed(tmpdir):
+    nwbfile_path = tmpdir / "test_make_or_load_nwbfile_empty_metadata.nwb"
+
+    with pytest.raises(ValueError):
+        with make_or_load_nwbfile(nwbfile_path=nwbfile_path, metadata=None, overwrite=True) as nwbfile:
+            nwbfile
```

Comment:
```suggestion
            pass
```

I think you can just do this?

Reply:
Yes both work, do you prefer the latter?

Reply:
I prefer the latter unless you want to do something with the object (print, assert something, etc)

Reply:
OK, let's change it. I am adding a changelog and changing to pass.

Reply:
Done.